src/main/java/com/myFirstProject/myFirstProject/service/AccountServiceImpl.java

```diff
@@ -178,7 +196,7 @@ public BigDecimal getTotalRevenue() {
 
     private void checkAccount(Account account, BigDecimal tacks) {
         if (account == null) {
-            throw new UserHaveNoAccount(String.format("User with id - %d have no account", account.getUser().getId()));
+            throw new UserHaveNoAccountException("User with  have no account");
```

Comment:
bad message: without information and with grammar mistakes!
"User with  have no account"